client/src/components/containers/CodeReviewMetricsContainers/CodeReviewFirstResponse/CodeReviewFirstResponseReviewersTable/CodeReviewFirstResponseReviewersTable.tsx

```diff
@@ -0,0 +1,79 @@
+import { NOT_AVAILABLE } from "src/constants/constants";
+import { IFetchedPullRequestVotesTimeline } from "src/services/api/api";
+import { getFormattedDateWithTime, getTimeFromSeconds, getTimeInDays, getTimeInSeconds } from "src/utils/utils";
+
+import styles from "./CodeReviewFirstResponseReviewersTable.module.scss";
+import { columns } from "./codeReviewFirstResponseReviewersTableConstants";
+
+interface ICodeReviewFirstResponseTable {
+  votesHistoryTimeline: IFetchedPullRequestVotesTimeline[];
+}
+
+export const CodeReviewFirstResponseReviewersTable = ({ votesHistoryTimeline }: ICodeReviewFirstResponseTable) => {
+  const reviewersFirstVotesArray = votesHistoryTimeline.reduce(
+    (acc: IFetchedPullRequestVotesTimeline[], vote: IFetchedPullRequestVotesTimeline) => {
+      if (!acc.find((v) => v.id === vote.id)) {
+        acc.push(vote);
+      }
+      return acc;
+    },
+    [],
+  );
+
+  return (
+    <div className={styles.tableContainer}>
+      <table className={styles.table} aria-label="sticky table">
+        <thead className={styles.tableHead}>
+          <tr>
+            {columns.map((column) => (
+              <th
+                key={column.id}
+                style={{
+                  width: column.width,
+                }}
+                className={styles.tableHeaderCell}
+              >
+                {column.label}
+              </th>
+            ))}
+          </tr>
+        </thead>
+
+        <tbody className={styles.tableBody}>
+          {reviewersFirstVotesArray.length ? (
+            reviewersFirstVotesArray.map((row) => {
+              const reviewerResponseTimeInSeconds =
+                row.timeOfVote && row.reviewerAddedTime
+                  ? getTimeInSeconds(row.timeOfVote, row.reviewerAddedTime)
```

Comment:
Can we move the safe checks inside the function so that consumers need not to aware of these